common/lib/dependabot/pull_request_creator.rb

```diff
@@ -187,17 +188,17 @@ def codecommit_creator
         base_commit: base_commit,
         credentials: credentials,
         files: files,
-        commit_message: message_builder.commit_message,
-        pr_description: message_builder.pr_message,
-        pr_name: message_builder.pr_name,
+        commit_message: message.commit_message,
+        pr_description: message.pr_message,
+        pr_name: message.pr_name,
         author_details: author_details,
         labeler: labeler,
         require_up_to_date_base: require_up_to_date_base?
       )
     end
 
-    def message_builder
-      @message_builder ||
+    def message
+      @message ||=
         MessageBuilder.new(
```

Comment:
Should this return `MessageBuilder.new(...).message`? Looks like that's currently unused

Reply:
It doesn't need to, since `Message` and `MessageBuilder` satisfy the same interface loose typing does the rest? 🦆==🦆 ?

I'm impartial but this suggestion appears twice so I'll take it as a Rubyism.